vkcs/compute/data_source_flavor.go

```diff
@@ -56,7 +57,7 @@ func DataSourceComputeFlavor() *schema.Resource {
 				Type:        schema.TypeInt,
 				Optional:    true,
 				ForceNew:    true,
-				Description: "The exact amount of RAM (in megabytes).",
+				Description: "The exact amount of RAM (in megabytes). Don't set ram, when min_ram is set.",
```

Comment:
probably add ConflictsWith min_ram?

Reply:
It would be a breaking change because users could write configs with both attributes set, and those configurations would not work if we added `ConflictsWith`.

Reply:
add a warning